examples/case-study-1/query-db.p6

```diff
@@ -3,3 +3,93 @@
 use Red;
 
 use lib <./lib>;
+use Present;
+use Attend;
+use Email;
+use Person;
+
+=begin comment
+
+Some queries:
+
+    1. alphabetical list of all contacts by last name, first name, and
+       include all known emails in alpha order
+    2. number of attendees per year
+    3. number of presenters per year
+    4. for a person (key), what years was the person an attendee or a presenter
+    5. a list of all known valid emails for upload to a mail server
+    6. list contacts with their keys
+
+=end comment
+
+my $usage = "Usage: $*PROGRAM <query number> [help | debug]";
+if !@*ARGS {
+    say $usage;
+    exit;
+}
+
+my $debug = 0;
+for @*ARGS {
```

Comment:
Would there be any value in making a bunch of ```multi MAIN``` with the constant parameters instead?

Reply:
Can you show me an example?

Reply:
```perl6
#| alphabetical list of all contacts by last name, first name, and
#| include all known emails in alpha order
multi MAIN(1) {
   # 1. alphabetical list of all contacts by last name, first name, and
   #    include all known emails in alpha order
}

#| number of attendees per year
multi MAIN(2) {
   # 2. number of attendees per year
}

#| number of presenters per year
multi MAIN(3) {
   # 3. number of presenters per year
}

#| for a person (key), what years was the person an attendee or a presenter
multi MAIN(4) {
   # 4. for a person (key), what years was the person an attendee or a presenter
   my $pkey = prompt "Enter person key (see query 6): ";
}

#| a list of all known valid emails for upload to a mail server
multi MAIN(5) {
   # 5. a list of all known valid emails for upload to a mail server
}

#| list contacts with their keys
multi MAIN(6) {
   # 6. list contacts with their keys
}
```

Reply:
```
$ perl6 query-db.p6 
Usage:
  query-db.p6 1 -- alphabetical list of all contacts by last name, first name, and include all known emails in alpha order
  query-db.p6 2 -- number of attendees per year
  query-db.p6 3 -- number of presenters per year
  query-db.p6 4 -- for a person (key), what years was the person an attendee or a presenter
  query-db.p6 5 -- a list of all known valid emails for upload to a mail server
  query-db.p6 6 -- list contacts with their keys
```

Reply:
https://docs.perl6.org/language/create-cli#index-entry-MAIN

Reply:
That's clear, thanks, but how would I add other arg handling like, say, 'debug'?

Reply:
```perl6
multi MAIN(1, :$debug) {...}
```

Reply:
so completed